src/components-examples/cdk/stepper/cdk-linear-stepper-with-form/cdk-linear-stepper-with-form-example.html

```diff
@@ -0,0 +1,17 @@
+<example-custom-linear-stepper [linear]="isLinear">
+  <cdk-step [stepControl]="firstFormGroup">
+    <label>Step 1 input</label>
+    <form [formGroup]="firstFormGroup">
+      <input placeholder="Input" formControlName="firstControl" required>
+    </form>
+  </cdk-step>
+  <cdk-step [stepControl]="secondFormGroup">
+    <label>Step 2 input</label>
+    <form [formGroup]="secondFormGroup">
+      <input placeholder="Input" formControlName="secondControl" required>
+    </form>
+  </cdk-step>
+</example-custom-linear-stepper>
+<button class="example-toggle-linear-button" (click)="isLinear = !isLinear">
```

Comment:
I'm not too familiar with the Angular style guide, but just curious if we generally recommend writing logic in the templates, or if this should be in the .ts file

Reply:
This is basically as complex of logic as we want in templates. I'm fine with this, but changing it to a function is also good.

Reply:
Wrote a function to toggle linearity and call that function on click.